static/app/components/onboardingWizard/sidebar.tsx

```diff
@@ -243,4 +272,15 @@ const TopRight = styled('img')`
   width: 60%;
 `;
 
-export default withApi(withOrganization(withProjects(OnboardingWizardSidebar)));
+function OnboardingWizardWrapper(props: Props) {
```

Comment:
Don't do this, just convert `usePersistedOnboardingState` to a functional component. it's almost there already.

Then you can also use `useApi` and potentially `useOrganization` and `useProjects`.

Reply:
Can we do the conversion in a separate PR and merge this as is with the wrapper

Reply:
https://github.com/getsentry/sentry/pull/34609 conversion here. Should make this a bit cleaner hopefully

Reply:
Thanks Evan!